tallycash/extension
# Remove Balance Reloader feature flag and functionality.

The button this feature flag enables has been off for months now - and was initially added as a temporary fix to address wallet balance issues which have since been resolved.

### To Test
- Load up the wallet
- Navigate to the `Wallet` page
- Page should render, buttons should be clickable, everything should work.
